src/test/java/io/nats/client/NKeyTests.java

```diff
@@ -27,8 +27,10 @@
 import static io.nats.client.support.Encoding.base32Decode;
 import static io.nats.client.support.Encoding.base32Encode;
 import static io.nats.client.utils.ResourceUtils.dataAsLines;
+import static io.nats.nkey.NKeyConstants.*;
```

Comment:
private and package scoped constants were removed from the class, so tests must rely on the extracted implementation.